test/jdk/java/util/Properties/PropertiesStoreTest.java

```diff
@@ -55,9 +55,7 @@
 public class PropertiesStoreTest {
 
     private static final String DATE_FORMAT_PATTERN = "EEE MMM dd HH:mm:ss zzz uuuu";
-    // use a neutral locale, since when the date comment was written by Properties.store(...),
-    // it internally calls the Date.toString() which always writes in a locale insensitive manner
```

Comment:
Instead of blindly removing the comment, I'd suggest reflecting the fact that `Date.toString()` uses `Locale.US` for time zone names.

Reply:
Thanks for your advice. The comment has been updated.